client/lib/i18n-utils/switch-locale.js

```diff
@@ -516,9 +516,6 @@ export function switchWebpackCSS( isRTL ) {
 	forEach( currentLinks, async ( currentLink ) => {
 		const currentHref = currentLink.getAttribute( 'href' );
 		const newHref = setRTLFlagOnCSSLink( currentHref, isRTL );
-		if ( currentHref === newHref ) {
```

Comment:
I think we can do the similar check inside `loadCSS` function. For example,

```js
function loadCSS( cssUrl, currentLink ) {
  return new Promise( ( resolve ) => {
    const isRTL =  i18n.isRTL();
    const isRTLLink = currentLink.url.endsWith( '.rtl.css' );
    if ( isRTL === isRTLLink ) {
      return resolve( null );
    }
    ...
  } );
}
```

So, even if we call the loadCSS multiple times, we won't inject the same link when the async function is called.

Next, we can use `newLink` to decide whether we have to remove the current link or not.

What do you think?